drivers/Kconfig

```diff
@@ -51,6 +51,7 @@ source "drivers/mdio/Kconfig"
 source "drivers/memc/Kconfig"
 source "drivers/mfd/Kconfig"
 source "drivers/mipi_dsi/Kconfig"
+source "drivers/mipi_dbi/Kconfig"
```

Comment:
dbi before dsi? Guess you wanted to make it alphabetical.

Reply:
Yes, unfortunately I mixed up my alphabet :)